src/index.js

```diff
@@ -36,6 +38,8 @@ const App = (appProps) => {
     );
   }
 
+  const commands = appKeyboardShortcuts.concat(ermDefaultKeyboardShortcuts);
```

Comment:
Lets use the spread operator here

```suggestion
  const commands =  [...appKeyboardShortcuts, ...ermDefaultKeyboardShortcuts]
```